.mcignore

```diff
@@ -0,0 +1,17 @@
+.git/
+node_modules/
+npm-debug.log
+.DS_Store
+*.sw[po]
+*.xpi
+*.pyc
+logs/
+dist/
+ping-centre/
```

Comment:
Just making sure, this is intended to ignore both `/ping-centre/` and `/test/unit/ping-centre`? I think it would be clearer if we were explicit about which things should be root vs matching anywhere. The `/locales/` fix avoided the prerendered missing files. As noted elsewhere, skipping all `.eslintrc.js` files happens to be fine too. But it's unclear if that was actually intentional or not.

Maybe have all the "match anywhere" patterns first then separately all the `/` root patterns (e.g., .git, node_modules, logs, dist)

Reply:
Yep, that sounds good

Reply:
I guess we do want to ignore ping centre tests if we aren't copy the source code